bundler/src/main/chart/templates/daemonset.yaml

```diff
@@ -40,12 +40,38 @@ spec:
           env:
             - name: "CONFIG_SERVER_URL"
               value: "http://{{ $.Values.global.services.configuration }}:8888/configserver"
+            - name: KEYSTORE_LOCATION
+              {{/*
+                 Set the keystore location from a secret if set, otherwise set from the pki destination and keystore name
+              */}}
+              {{- if $.Values.global.secrets.keystore.path -}}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Values.global.secrets.keystore.name }}
+                  key: {{ $.Values.global.secrets.keystore.path }}
+                  optional: false
+              {{- else -}}
+              value: "file://{{ $.Values.global.volumes.pki.destination }}/{{ default $.Values.global.volumes.pki.keystore $.Values.keystore }}"
```

Comment:
Does it make sense to specify a default for the second half of this path? (either .Values.global.volumes.pki.keystore or .Values.keystore), in either the umbrella or individual charts?  If we make the if statement evaluate false, this just shows a directory currently.

Reply:
good point, will update